src/utilities/toPascalCase.ts

```diff
@@ -0,0 +1,26 @@
+import {upperCaseFirstCharacter} from './upperCaseFirstCharacter'
+
+export const toPascalCase = (string: string | string[]) => {
+  if (!Array.isArray(string)) {
+    string = [string]
+  }
+  // match unsupported characters
+  const regex = /[^a-zA-Z0-9]+/g
+  // replace any non-letter and non-number character and split into word array
+  const stringArray = string
+    .filter(part => part !== '@')
+    .filter(Boolean)
```

Comment:
Nit, but you could combine these

Reply:
Yes, but I feel it is easier to read this way and performance is not really a bottle-neck.

You mean like this, right?

```js
 .filter(part => part !== '@' && Boolean(part))
````

Reply:
Actually ended up refactoring this a bit